### PR TITLE
FP added favorites add/remove logic

### PR DESCRIPTION
**Pull request by AsyyaT**

### CODE REVIEW

File: src/js/favorites/favorites.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const getFavorites = () => {
+  const favorites = localStorage.getItem('favorites');
+  return favorites ? JSON.parse(favorites) : [];
+}
+
+export const saveFavorites = favorites =>{
```
Comment (snegurova) [inline src/js/favorites/favorites.js]: The code should be formatted by prettier

File: src/js/favorites/favorites.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  localStorage.setItem('favorites', JSON.stringify(favorites));
```
Comment (snegurova) [inline src/js/favorites/favorites.js]: We will not have favorites as array to save, we will save one item, which we will need to add to the existed array or create new local storage item and push only this one item to array

File: src/js/favorites/favorites.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+export const toggleFavorite = exerciseId => {
```
Comment (snegurova) [inline src/js/favorites/favorites.js]: let's have add and remove, for direct simple logic

Comment (AsyyaT) [reply src/js/favorites/favorites.js]: sure

**Pull request closed without merging**